sql/migrations/20170614155841_uniqueDelegatesConstraint.sql

```diff
@@ -0,0 +1,9 @@
+/*
+ * Recreate 'delegates' table to get rid of duplicated entries
```

Comment:
Comment is wrong now. Should say "setting unique constraints on delegates table"